### ISSUE TITLE
tap_action always toggles menu

### ISSUE THREAD

**Issue by Villhellm** [iantrich/radial-menu#8]
config:
```
      - type: 'custom:radial-menu'
        entity: light.living_room
        style:
          left: 33%
          top: 27%
        tap_action:
          action: call-service
          service: light.toggle
          service_data:
            entity_id: light.living_room
        hold_action:
          action: toggle-menu
        items: ...
```

When trying to override the tap_action the menu still opens instead of just running the tap_action. The light toggles, but so does the menu. I'm trying to only have the menu open on hold. Am I missing something?

**Comment by Villhellm** [comment]
Setting `default_dismiss` to false "fixes" that problem, but that still seems a bit strange that it needs to be set to handle the initial tap_action

**Comment by iantrich** [comment]
should be fixed in 1.4.5
